composer.json

```diff
@@ -21,7 +21,7 @@
         }
     ],
     "require": {
-        "php": "^5.6",
+        "php": "^5.5",
```

Comment:
Yeah, thats been my intention all along. I wanted to build v1 for php5.6 first and then branch off and refactor/rebuild v2 for php7. I've got an un-pushed branch that does this, I'm just waiting to tag v1 before I rebase and push.